src/NuGet.Core/NuGet.Packaging.Core/IPackageCoreReader.cs

```diff
@@ -58,6 +59,7 @@ public interface IPackageCoreReader : IDisposable
             string destination,
             IEnumerable<string> packageFiles,
             ExtractPackageFileDelegate extractFile,
+            ILogger logger,
```

Comment:
I'll look into that, it might be too limiting, like if there are multiple users of this interface and they each want their own logging.

---
In reply to: [59910966](https://github.com/NuGet/NuGet.Client/pull/504#discussion_r59910966) [](ancestors = 59910966)